.github/workflows/clippy.yml

```
@@ -0,0 +1,26 @@
+on: [push]
+
+name: Clippy
+
+jobs:
+  clippy_check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          components: clippy
+          override: true
+
+      - name: Install gtk3 and dependencies.
+        run: sudo apt update && sudo apt install libcurl4-openssl-dev libelf-dev libdw-dev binutils-dev libgtk-3-dev libmount-dev libsoup2.4-dev libxtst-dev at-spi2-core openbox libxdo-dev
```

```suggestion
        run: |
          sudo apt update
          sudo apt install \
            libcurl4-openssl-dev \
            libelf-dev \
            libdw-dev \
            binutils-dev \
            libgtk-3-dev \
            libmount-dev \
            libsoup2.4-dev \
            libxtst-dev \
            at-spi2-core \
            openbox \
            libxdo-dev
```

Much easier to edit like this. :)

---

Ooh, nice, I didn't know you could do that. Thank you!